.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
           RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${new_fix_notes}"
 
```

Comment:
- It is recommended to use meaningful variable names instead of generic ones like 'RELEASE_NOTES' and 'updated_content'.